src/desktop/apps/artwork/components/commercial/templates/questions.jade

```diff
@@ -1,13 +1,12 @@
-- isBuyNowWork = artwork.is_acquireable && (newBuyNowFlow && artwork.partner.type != 'Auction House')
+- isBuyNowWork = artwork.is_acquireable
```

Comment:
Not clear about this one... 

I think the difference will only be for 'buy now' works from auction partner.

Currently those works will display 'collector FAQ' only link (https://www.artsy.net/artwork/5bbe518344110a3ccf77c267)

I believe with this change they will start displaying standard `buy now links` - buy now FAQ + contact a specialist link. Similar to this: https://www.artsy.net/artwork/josef-albers-homage-to-the-square-87.

Is that what we want?

Reply:
Oh actually good point, it looks like I missed this. We probably want to keep the same behavior here. Thanks for the 👀 @oxaudo and @mzikherman !

Reply:
The problem with this page is that it has so many if/else cases that I'm not sure anybody knows 100% what it's supposed to be. Maybe worth checking what links should actually be displayed for 'buy now' auction work. Maybe @joanna20 or Melanie could be consulted here.

Also not even clear if the artwork from auction can have regular bidding and available to buy now. What links should be there in that case?

Reply:
Auction artworks _can_ have both bidding + buy now! But i'll follow up and at the very least keep existing  behavior.

Reply:
Just spoke with Joanna and the desired behavior is:
- For an ecommerce work that's by an auction partner (but that is not _biddable_), we want to show the new FAQ and the "ask a specialist" link (so it will have the same behavior as an ecommerce work by a non-auction partner).
- For an ecommerce work that's also biddable in an auction, we actually don't even show this component so it's not an issue here.

Reply:
Thanks for following up with this @sweir27. I think it's good to simplify things here.

For the new responsive page - we need to rebuild all of that. So great to get some clarity about link expectations for each type of works. Hopefully @steven will put those in one issue once we get to that point:)